src/app/utils/date.ts

```diff
@@ -0,0 +1,3 @@
+export function secondsToDate(seconds: number) {
```

Comment:
Add return type

Reply:
I think it is not needed, TypeScript use type inference and if I remeber it right we shouldn't put return type if we don't have to.